changes/ticket32604

```diff
@@ -0,0 +1,7 @@
+  o Minor features (onion services):
+    - Version 3 onion services can now use HiddenServiceExportRendPoint
+      to get the IP and port of the rendezvous point they are connecting.
```

Comment:
I think I'd have written "connecting to" here?

Reply:
Fixed in `f38218a`.